### PR TITLE
Fixed bug in cloneWithRef.js

### PR DESCRIPTION
**Pull request by standuprey**
Fixed bug in cloneWithRef.js that prevented to do something like 

    render() {
      const { isDragging, connectDropTarget, connectDragSource, isOver, text } = this.props;
      return connectDropTarget(connectDragSource(
        <div className="Card" style={{ opacity: isDragging ? 0.1 : 1 }}>
          {text}
        </div>
      ));
    }

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,10 +50,14 @@
   "devDependencies": {
     "animation-frame": "^0.2.4",
     "autoprefixer": "^5.0.0",
-    "babel": "^6.5.2",
+    "babel-cli": "^6.24.1",
```
Comment (NekR) [inline package.json]: Wh did you change this?

File: package.json [hunk 2]
```diff
@@ -50,10 +50,14 @@
...
     "babel-core": "^6.21.0",
     "babel-loader": "^6.2.10",
     "babel-plugin-transform-decorators-legacy": "^1.3.4",
+    "babel-plugin-transform-class-properties": "6.24.1",
+    "babel-plugin-transform-react-jsx": "6.24.1",
+    "babel-preset-es2015": "6.24.1",
+    "babel-preset-stage-0": "6.24.1",
```
Comment (NekR) [inline package.json]: and added these?

Comment (standuprey) [reply package.json]: I did, it wouldn't build without this for me, I suppose you had some packages globally installed?

Now, the tests are still failing for me, but at least it builds.

I can make a PR with only the change in cloneWithRefs if need be

**Comment by standuprey** [comment]
I updated the PR without the added dependencies

**Review by developit — APPROVED** [review]
LGTM

**Comment by NekR** [comment]
Yeah, yeah, I'll merge this.

@developit is it possible to request non-project related people as reviewers or I made you a collaborator and forgot about it? 😄

**Comment by developit** [comment]
no clue lol. maybe I added myself?

**Comment by standuprey** [comment]
A merge/release would be lovely

**Comment by NekR** [comment]
Thanks! I'll make a release.

@developit no idea, weird!

**Comment by NekR** [comment]
Published to npm.

@standup75 btw, was the tests failing before this change or what exactly?

**Comment by standuprey** [comment]
I realized that building a proof of concept with this for our project. https://github.com/standup75/preact-dnd-demo

**Pull request merged**